src/sentry/eventstream/kafka/backend.py

```diff
@@ -275,13 +286,13 @@ def run_post_process_forwarder(
         concurrency: int,
         initial_offset_reset: Union[Literal["latest"], Literal["earliest"]],
         strict_offset_reset: bool,
-    ):
+    ) -> None:
         logger.debug(f"Starting post process forwarder to consume {entity} messages")
-        if entity == PostProcessForwarderType.TRANSACTIONS:
+        if entity == "transactions":
```

Comment:
Why change this to raw strings? It seems like it would be better to keep the `PostProcessForwarderType` and have the `run_post_process_forwarder` function take a `PostProcessForwarderType` as an argument.

Reply:
Yeah, I'd prefer we don't change these if possible